app/views/finance/lead_providers/show.html.erb

```diff
@@ -0,0 +1,98 @@
+<% content_for :title, "Lead Provider Payment Breakdown" %>
+<%
+  def band_index_to_heading(number)
+    ("A".."Z").to_a[number]
+  end
+%>
+
+<h1 class="govuk-heading-l"><%= @ecf_lead_provider.name %></h1>
+
+<table class="govuk-table">
+  <caption class="govuk-table__caption govuk-table__caption--l">ECF Payment breakdown</caption>
+  <tbody class="govuk-table__body">
+      <tr class="govuk-table__row">
+        <th class="govuk-table__header govuk-!-width-one-half">Milestone</th>
+        <td class="govuk-table__cell" data-test="lead-provider-name">Started</td>
+      </tr>
+      <tr class="govuk-table__row">
+        <th class="govuk-table__header">Recruitment target</th>
+        <td class="govuk-table__cell" data-test="lead-provider-name"><%= @ecf_lead_provider.call_off_contract[:recruitment_target] %></td>
+      </tr>
+      <tr class="govuk-table__row">
+        <th class="govuk-table__header">Current ECTs</th>
+        <td class="govuk-table__cell" data-test="lead-provider-name"><%= @total_ect %></td>
+      </tr>
+      <tr class="govuk-table__row">
+        <th class="govuk-table__header">Current mentors</th>
+        <td class="govuk-table__cell" data-test="lead-provider-name"><%= @total_mentors %></td>
+      </tr>
+      <tr class="govuk-table__row">
+        <th class="govuk-table__header">Current participants</th>
+        <td class="govuk-table__cell" data-test="lead-provider-name"><%= @total_participants %></td>
+      </tr>
+  </tbody>
+</table>
+
+<table class="govuk-table">
+  <caption class="govuk-table__caption govuk-table__caption--l">Service Fee</caption>
+  <thead class="govuk-table__head">
+    <tr class="govuk-table__row">
+      <th class="govuk-table__header">Band</th>
+      <th class="govuk-table__header">Number of Participants</th>
+      <th class="govuk-table__header">Payment amount per person</th>
+      <th class="govuk-table__header">Payment amount monthly</th>
+    </tr>
+  </thead>
+  <tbody class="govuk-table__body">
+    <% @breakdown[:service_fees].each_with_index do |service_fee, i| %>
+    <tr class="govuk-table__row">
+      <td class="govuk-table__cell"><%= band_index_to_heading(i) %></td>
+      <td class="govuk-table__cell"><%= @ecf_lead_provider.call_off_contract.bands[i].number_of_participants_in_this_band(@ecf_lead_provider.call_off_contract[:recruitment_target]) %></td>
```

Comment:
Should `service_fee` include this in its output?

Reply:
Yeah I think this view knows far too much about the internals of things further down.

Reply:
Numbers of participants in a band is not a service fee. It feels more like a rename/refactor to get the correct structure of things when we decide what that is.

Reply:
Sure, numbers of participants is not a service fee, in the same way as email is not a user. 

But just like user fields are strongly related to user, I reckon number of participants is strongly related to the service fee. I mean, we surface `service_fee_total`, and `service_fee_per_participant` - if we really wanted to we could divide one by the other and get total participants this way. Likewise for band name - every method in the service fee calculator depends on the band, so I don't see much harm in surfacing it.

I think we can stick a comment in the service fee calculator saying something like "TODO: Figure out if this needs to be renamed to something more accurate", but I would not want that to be the justification we use for putting
```
@ecf_lead_provider.call_off_contract.bands[i].number_of_participants_in_this_band(@ecf_lead_provider.call_off_contract[:recruitment_target])
```
in our view - that seems much worse than making `service_fee` an imprecise name.